crates/shared/src/event_storing_helpers.rs

```diff
@@ -11,8 +17,8 @@ use {
     number::conversions::u256_to_big_decimal,
 };
 
-pub fn create_quote_row(data: QuoteData) -> DbQuote {
-    DbQuote {
+pub fn create_quote_row(data: &QuoteData) -> Result<DbQuote> {
```

Comment:
What's the reason to take by reference now?

Reply:
That's right, pass by value is ok here as `create_quote_row` function consumes the data.